internal/authproxy/proxy/manager.go

```diff
@@ -171,6 +172,26 @@ HealthCheckLoop:
 	return localPort, nil
 }
 
+type RunningProxyInfo struct {
+	PortName   string
+	TargetPort int
+	LocalPort  int
+}
+
+func (m *Manager) GetRunningProxies() (runningProxies []RunningProxyInfo) {
```

Comment:
Is this function used in production code?

Reply:
No. Used only in test code.
I think it is a function that can be used as a product code.

Reply:
I think it is better not to include unused code even though it may use in the future.
Also it is different implementation from GetRunningProxy so it may be confusing next contributors

Reply:
🆗
So what kind of implementation would you like?

Reply:
I will write the suggestion, please wail.

Reply:
created https://github.com/oruharo/cosmo/pull/33

Reply:
Thank you!  I merged https://github.com/oruharo/cosmo/pull/33.